java/HealthVault Mobile/HVSample/src/main/res/layout/add_medication.xml

```diff
@@ -4,205 +4,157 @@
 	android:layout_height="match_parent">
 
 	<!-- Name Labels/Text -->
-
 	<TextView
 		android:id="@+id/name_label"
-		android:layout_width="@dimen/width_69dp"
-		android:layout_height="@dimen/height_20dp"
-		android:layout_marginTop="@dimen/margintop_20dp"
-		android:layout_marginLeft="@dimen/marginleft_20dp"
+		android:layout_width="match_parent"
+		android:layout_height="@dimen/add_medicationlabels_hight"
+		android:layout_marginTop="@dimen/add_medicationlabels_top"
+		android:layout_marginLeft="@dimen/add_medicationlabels_marginleft"
+		android:textSize="@dimen/add_medicationlabels_textsize"
 		android:textColor="@color/old_lavender"
-		android:textSize="@dimen/textsize_15sp"
 		android:text="@string/name_label" />
-
-	<EditText
+	<TextView
 		android:id="@+id/name_text"
 		android:layout_width="match_parent"
-		android:layout_height="@dimen/height_48dp"
-		android:layout_below="@+id/name_label"
-		android:layout_marginTop="@dimen/margin_10dp"
-		android:layout_marginLeft="@dimen/marginleft_20dp"
-		android:layout_marginRight="@dimen/marginright_20dp"
-		android:background="@color/Isabelline"
-		android:ems="10"
-		android:inputType="text"
-		android:paddingLeft="@dimen/padding_20dp"
-		android:text="@string/sample_name_text"
-		android:textSize="@dimen/textsize_15sp"/>
+		android:layout_height="@dimen/add_medicationlabels_hight"
+		android:textSize="@dimen/add_medicationlabels_textsize"
+		android:paddingLeft="@dimen/add_medicationtext_padding"
+		android:textColor="@color/pine_green"
+		android:layout_below="@+id/name_label"/>
```

Comment:
This layout root element is a RelativeLayout yet each child is just using layout_below to layout below the item above it. Instead, use a LinearLayout with vertical orientation as the root element. Then we won't need to add layout_below to each and every child item which will make things way easier.